### ISSUE TITLE
loop error

### ISSUE THREAD

**Issue by jung27** [TheLimeGlass/Skellett#160]
this is my code
```
command /asd:
	trigger:
		loop 3 times:
			send "asd"
```

but it make an this error
```
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! [Skript] Severe Error:
[10:07:19 ERROR]: #!#! Could not load asd.sk
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Something went horribly wrong with Skript.
[10:07:19 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[10:07:19 ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[10:07:19 ERROR]: #!#! Here is full list of them:
[10:07:19 ERROR]: #!#! SkBee v1.15.1 (https://github.com/ShaneBeee/SkBee) Skellett v2.0.3 (https://forums.skunity.com/resources/skellett.24/) SkQuery v4.1.6
[10:07:19 ERROR]: #!#! We could not identify which of those are specially related, so this might also be Skript issue.
[10:07:19 ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[10:07:19 ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[10:07:19 ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[10:07:19 ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[10:07:19 ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Stack trace:
[10:07:19 ERROR]: #!#! ch.njol.skript.SkriptAPIException: No class info found for citizen
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.registrations.Classes.getClassInfo(Classes.java:263)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.patterns.TypePatternElement.fromString(TypePatternElement.java:92)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:132)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:48)
[10:07:19 ERROR]: #!#!     at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1220)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1301)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:226)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:179)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Condition.parse(Condition.java:77)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.sections.SecConditional.init(SecConditional.java:83)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Section.init(Section.java:69)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:257)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:179)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Section.lambda$parse$0(Section.java:157)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Section$SectionContext.modify(Section.java:190)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Section.parse(Section.java:156)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.loadItems(ScriptLoader.java:1127)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.command.Commands.loadCommand(Commands.java:489)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:707)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.lambda$loadScripts$4(ScriptLoader.java:525)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.lambda$makeFuture$2(ScriptLoader.java:417)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.makeFuture(ScriptLoader.java:433)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:524)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.reloadScript(ScriptLoader.java:1060)
[10:07:19 ERROR]: #!#!     at Skript.jar//ch.njol.skript.SkriptCommand.onCommand(SkriptCommand.java:171)
[10:07:19 ERROR]: #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[10:07:19 ERROR]: #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[10:07:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchCommand(CraftServer.java:905)
[10:07:19 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchServerCommand(CraftServer.java:868)
[10:07:19 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.be(DedicatedServer.java:510)
[10:07:19 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:484)
[10:07:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1444)
[10:07:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1218)
[10:07:19 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[10:07:19 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Version Information:
[10:07:19 ERROR]: #!#!   Skript: 2.6.1 (latest)
[10:07:19 ERROR]: #!#!     Flavor: skriptlang-github
[10:07:19 ERROR]: #!#!     Date: 22:18:42.554713200
[10:07:19 ERROR]: #!#!   Bukkit: 1.18.2-R0.1-SNAPSHOT
[10:07:19 ERROR]: #!#!   Minecraft: 1.18.2
[10:07:19 ERROR]: #!#!   Java: 17.0.1 (Java HotSpot(TM) 64-Bit Server VM 17.0.1+12-LTS-39)
[10:07:19 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Server platform: Paper
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Current node: loop 3 times: (dvc.sk, line 3)
[10:07:19 ERROR]: #!#! Current item: null
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Thread: Server thread
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! Language: english
[10:07:19 ERROR]: #!#! Link parse mode: DISABLED
[10:07:19 ERROR]: #!#!
[10:07:19 ERROR]: #!#! End of Error.
[10:07:19 ERROR]: #!#!
```
I've confirmed that this error is being caused by the Skellet addon.

**Comment by organizatsiya** [comment]
Hello

I have the same problem with Skelett for "loop"

The skript :

```
command /Loop <integer>:
    trigger:
        loop arg-1 times:
            broadcast "%arg-1%"
```
>.... [21:11:30 ERROR]: #!#! [Skript] Severe Error:
>.... [21:11:30 ERROR]: #!#! Could not load test.sk
>.... [21:11:30 ERROR]: #!#! Current node: loop arg-1 times: (test.sk, line 3)

whereas if I disable Skelett, the skript works

Skelett-Legacy-2.0.3
Skript 2.6.1
Paper 1.18.2
Java17

**Comment by TheLimeGlass** [comment]
Duplicate and fixed in 2.0.4 https://github.com/TheLimeGlass/Skellett/issues/167
